aks.tf

```diff
@@ -38,6 +38,15 @@ resource "azurerm_kubernetes_cluster" "k8s_cluster" {
   tags {
     Environment = "${var.environment}"
   }
+	
+	role_based_access_control {
```

Comment:
Må formateres likt som resten av koden, ref. tabs vs. spaces. En god praksis for Terraform er å kjøre `terraform fmt` før commit 🙂

Reply:
Ja, har vanligvis tabstop=2 og expandtab satt, men den siste manglet og da så alt likt ut i klienten min